client/src/components/Layout/Header.js

```diff
@@ -43,98 +34,97 @@ const useStyles = makeStyles((theme) => ({
       minHeight: "45px",
     },
   },
-  content: {
-    display: "flex",
-    justifyContent: "space-between",
-    alignItems: "center",
-    paddingRight: "1em",
-  },
   logo: {
     maxWidth: "175px",
-    height: "100%",
-    width: "100%",
-    margin: ".5rem",
+    maxHeight: "48px",
+    margin: "4px 4px 0 4px",
     "&:hover": {
       filter: "brightness(1.2)",
     },
+    [theme.breakpoints.down("xs")]: {
+      maxHeight: "36px",
+      margin: "4px 4px 0 8px",
+    },
   },
   logoStacked: {
     height: "40px",
     margin: "auto .5rem auto 0.5rem",
     "&:hover": {
       filter: "brightness(1.2)",
     },
-    // [theme.breakpoints.down("sm")]: {
-    //   height: "50px",
-    // },
+  },
+  content: {
+    display: "flex",
+    flexGrow: 2,
+    justifyContent: "space-between",
+    alignItems: "center",
+    margin: "0px 24px",
   },
   tagline: {
-    color: theme.palette.primary.main,
+    color: theme.palette.primary.dark,
+    display: "inline-block",
     fontWeight: "bold",
     fontSize: "14px",
     lineHeight: "1.2",
-    textAlign: "center",
-    flexGrow: 1,
+    flexGrow: 2,
     flexShrink: 1,
     fontFamily: `Helvetica, Arial, "Lucida Grande", sans- serif`,
     [theme.breakpoints.up("sm")]: {
       display: "none",
     },
   },
   username: {
-    color: "#1b1b1b",
+    color: theme.palette.primary.dark,
+    display: "inline-block",
     fontStyle: "italic",
     lineHeight: "1.5",
-    flexBasis: "1em",
-    textAlign: "right",
     flexGrow: 1,
     fontFamily: `"HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans- serif`,
   },
 }));
 
 export default function Header(props) {
-  const { user, setUser, tenantId, setToast } = props;
+  const { user, setUser, tenantId, setToast, taglineText } = props;
   const classes = useStyles();
-  const taglineText = isMobile()
-    ? "Locate free food in " + tenantName
-    : "Locate free food in " + tenantName;
+  const imageType = logoPaths
+    ? logoPaths[tenantId].default.split(".").pop()
+    : "unknown";
 
   return (
     <>
       <AppBar position="sticky" className={classes.headerHolder}>
         <Toolbar className={classes.header}>
+          <div>
+            <a href="/">
+              <img
+                src={logoPaths[tenantId] ? logoPaths[tenantId].default : logo}
+                className={classes.logo}
+                alt="logo"
+                style={
+                  imageType === "svg" ? { width: "100%", height: "100%" } : {}
+                }
+              />{" "}
+            </a>
+          </div>
           <div className={classes.content}>
-            <div>
-              {isMobile() ? (
```

Comment:
Can't remove the stacked logo from the mobile view (at least for LA and other non-HI tenants) without further discussion from the LA design team and Product Managers.

Reply:
@entrotech should be fixed